lib/runner/VirtualGoogleAssistantInvoker.js

```diff
@@ -39,7 +43,7 @@ module.exports = class VirtualGoogleAssistantInvoker extends Invoker {
         try {
             await this._virtualGoogleAssistant.stopExpressServer();
         } catch(e) {
-            if (e.message !== "This instance is not using express") {
+            if (e !== "This instance is not using express" && e.message !== "This instance is not using express") {
```

Comment:
maybe express throws a custom error here to used instead message

Reply:
We don't have a way from virtual google assistant to see if it's using express, and we need to start and stop the server if that's the case. This is the way to do it safely.
I update it since there were some cases where the error was being returned directly as a string during tests.